src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.Product.APPLE;
+import static core.basesyntax.model.Product.BANANA;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class TransactionParserImplTest {
+    private static TransactionParserImpl transactionParser;
+
+    @BeforeAll
+    static void beforeAll() {
+        transactionParser = new TransactionParserImpl();
+    }
+
+    @DisplayName("Check correct input common data")
+    @Test
+    void getTransactions_validInput_ok() {
+        List<String> inputList = List.of(
```

Comment:
Let's better use concrete naming instead of abstract `list`.

```suggestion
        List<String> inputTransactions = List.of(
```